draft-ietf-quic-datagram.md

```diff
@@ -242,9 +242,12 @@ priorities to DATAGRAM frames with respect to each other and to QUIC streams.
 
 Although DATAGRAM frames are not retransmitted upon loss detection, they are
 ack-eliciting ({{!RFC9002}}). Receivers SHOULD support delaying
-ACK frames (within the limits specified by max_ack_delay) in reponse to receiving
-packets that only contain DATAGRAM frames, since the timing of these
-acknowledgements is not used for loss recovery.
+ACK frames (within the limits specified by max_ack_delay) in response to
+receiving packets that only contain DATAGRAM frames, since the sender takes no
+action if these packets are temporarily unacknowledged. Receivers SHOULD
```

Comment:
We're not adding new normative behavior for acknowledgements, so I think we should just state the fact of how QUIC will behave here.

```suggestion
action if these packets are temporarily unacknowledged. Receivers will
```